src/__tests__/role-helpers.js

```diff
@@ -132,13 +132,16 @@ test('getRoles returns expected roles for various dom nodes', () => {
   })
 })
 
-test('logRoles logs expected roles for various dom nodes', () => {
-  expect.addSnapshotSerializer(jestSerializerAnsi)
-
+test('logRoles calls console.log with output from prettyRoles', () => {
   const {section} = setup()
-  const output = logRoles(section)
+  expect.addSnapshotSerializer(jestSerializerAnsi)
 
-  expect(output).toMatchSnapshot()
+  jest.spyOn(console, 'log')
+  logRoles(section)
+  // eslint-disable-next-line no-console
+  expect(console.log.mock.calls).toHaveLength(1)
```

Comment:
This should probably be swapped with `expect(console.log).toHaveBeenCalledTimes(1)` to improve any error message that may occur.

Reply:
Done [5fefa1](https://github.com/testing-library/dom-testing-library/pull/312/commits/5fefa10d167518fa396538a09ef428b81f2ce45d)